tests/offload_utils.hpp

```diff
@@ -1,6 +1,13 @@
 
 namespace {
 bool dispatch_host = true;
+
+#define __MDSPAN_DEVICE_ASSERT_EQ(LHS, RHS) \
```

Comment:
Double underscore prefix doesn't sound like a good idea.  I don't think it is legal.

Reply:
OK but then we need to rename a few others too. We can do that in a separate PR.